README.md

```diff
@@ -11,187 +11,32 @@ The Customers project focuses on solving authentic identification of humans who
 
 [FFIEC Bank Secrecy Act - Customer Identification Program](https://www.fdic.gov/regulations/examinations/bsa/ffiec_cip.pdf)
 
-Docs: [Project](https://github.com/moov-io/customers/tree/master/docs/) | [API Endpoints](https://moov-io.github.io/customers/api/) | [Admin API Endpoints](https://moov-io.github.io/customers/admin/)
+Docs: [docs](docs/README.md) | [API Endpoints](https://moov-io.github.io/customers/api/) | [Admin API Endpoints](https://moov-io.github.io/customers/admin/)
 
 ## Project Status
 
 Moov Customers is under active development, so please star the project if you are interested in its progress. We are developing an extensible HTTP API for interactions along with an OpenAPI specification file for generating clients for integration projects.
 
-### Running Locally
+## Getting Started
```

Comment:
Can we copy the docker-compose commands here? That way someone can stay on the readme and try out Customers locally.